system_model/uc-docs.puml

```diff
@@ -0,0 +1,65 @@
+@startuml
+left to right direction
+skinparam linetype polyline
+
+frame f1 as "uc [site] Project Tetra Docs [Use Cases and Associated Stakeholders] " {
```

Comment:
This shows in your graphic, is that intentional? If so, it's not clear (to me) what it tells me.

Reply:
It's a SysML convention. I'll try and address it in a separate PR that closes #23